bpf/lib/common.h

```diff
@@ -850,7 +861,8 @@ struct ct_entry {
 	      dsr:1,
 	      from_l7lb:1, /* Connection is originated from an L7 LB proxy */
 	      auth_required:1,
-	      reserved:6;
+	      from_tunnel:1, /* Connection is over tunnel */
```

Comment:
@YutaroHayakawa I noticed that we are missing some visibility information when we are dumping the CT map. Shouldn't we add a "FromTunnel" in [here](https://github.com/cilium/cilium/blob/1f7ed72e6b132202caaa085d6774bb62f58ca7b4/pkg/maps/ctmap/types.go#L555-L594)?